client/components/Account/AddCredentialForm.vue

```diff
@@ -12,13 +12,17 @@ export default {
       method: 'POST',
       setAccount: true,
       fields: [
-        {id: 'username', label: 'Username'},
-        {id: 'password', label: 'Password', type: 'password'}
+        {id: 'username', label: 'New Username'},
+        {id: 'password', label: 'New Password', type: 'password'}
       ],
-      title: 'Create a user and add to the shared account',
+      validators: {
+        username: this.$helpers.validators.username,
+        password: this.$helpers.validators.password,
+      },
+      title: 'Add new person to the shared account',
```

Comment:
Does adding a credential still mean creating a user account? If so, I think it makes more sense to have the title as "Create and add a user to the shared account" or "Create a user account to add to the shared account" or something along those lines

Reply:
It means the same thing it always did, but I was trying to use less jargon because people were saying that "user" and "account" isn't as clear to this audience

Reply:
Sounds reasonable, I meant to emphasize more the create and add aspect than the user vs person aspect